mis_builder/models/mis_report_instance.py

```diff
@@ -406,7 +407,12 @@ def _get_additional_query_filter(self, query):
         Returns an Odoo domain expression (a python list)
         compatible with the model of the query."""
         self.ensure_one()
-        return []
+        domain = []
+        if (company_field := query.sudo().company_field_id) and (
+            instance_companies := self.report_instance_id.query_company_ids
```

Comment:
I think `query_company_ids` cannot be empty. And if it was, it would be surprising if the company field on the query was ignored. So may be we could simply `assert self.report_instance_id.query_company_ids` if `company_field_id` is set.